http/stylesheets/seasons/_standings.scss

```diff
@@ -10,3 +10,9 @@
     }
   }
 }
+
+span.without-forfeits {
+  font-size: 0.85rem;
+  font-style: italic;
+  color: #9ba0ab;
```

Comment:
i thought about that but then all the colours in `common` are named with very specific names like `$winner-cell-bg-color` which seem very designed-for-purpose rather than for reuse. i can move it there anyway but putting an only-useful-in-one-file colour definintion into a different file seems low value

Reply:
One really good thing about keeping all the colors in one file is that it makes it easier to manage the colors your site and either add support for multiple themes or change the color scheme entirely. Also helps someone really good at UX _(not me)_ easily fiddle around on a single file to make the site look better.

The general way I've seen people manage their colors is to have generic color names at the top of a common file like `color-base-grey-light`, `color-primary-navy-blue`, `color-cta-aquanarine` etc. and then have something similar to what I have in that colors file - `$obsolete-qualifier-row-color: $color-cta-dim-red` or something similar. I wanted to go back at some point and fix this file to be more like that but that's probably really low on the priority scale.

Reply:
yeah, having generic names in a common place makes sense (thats sort of what tailwind gives you for free, one of its nice features). i guess supporting multiple colour schemes sounds like a nice ideal but i suspect that's never going to happen in practice. i moved it anyway though *shrug*